prow/plugins/golint/suggestion/golint_suggestion.go

```diff
@@ -92,6 +95,38 @@ func fixStutter(p lint.Problem) string {
 	return suggestion
 }
 
+func fixErrorf(p lint.Problem) string {
+	matches := lintErrorfRegex.FindStringSubmatch(p.Text)
+	if len(matches) != 3 {
+		return ""
+	}
+	parameterText := ""
+	count := 0
+	parameterTextBeginning := "errors.New(fmt.Sprintf("
+	for _, char := range p.LineText[strings.Index(p.LineText, parameterTextBeginning)+len(parameterTextBeginning):] {
```

Comment:
`strings.Index` could return -1 if there is a bug in this code and that would cause `hook` to crash. It would be better to check that the index is non-negative and log a programmer error and return `""` if it is negative.